config/canonical.json

```
@@ -30760,7 +30760,9 @@
     "tags": {
       "brand": "Palí",
       "name": "Palí",
-      "shop": "supermarket"
+      "shop": "supermarket",
+      "brand:wikidata": "Q1064887",
+      "brand:wikipedia": "en:Walmart_de_México_y_Centroamérica"
```

remove the `_` - thanks!
```suggestion
      "brand:wikipedia": "es:Walmart de México y Centroamérica"
```

---

Also, I don't see any mention of the "Palí" brand on the `en:` page, so I changed it to the `es:` page which mentions it.